core-bundle/src/Resources/contao/modules/Module.php

```diff
@@ -281,6 +281,7 @@ protected function renderNavigation($pid, $level=1, $host=null, $language=null)
 		$objTemplate->type = \get_class($this);
 		$objTemplate->cssID = $this->cssID; // see #4897
 		$objTemplate->level = 'level_' . $level++;
+		$objTemplate->module = $this->objModel; // see #155
```

Comment:
Why not pass `$this`?

Reply:
@Defcon0's request was to access the module model in the navigation templates. We might as well pass `$this` – I don't care.

Reply:
I would pass the module itself. That's what the attribute is named after (otherwise is should be `moduleModel`), and it allows to retrieve the same data and more.

Reply:
More data that nobody needs ATM. So it would bloat the object for no reason. 🤷‍♂

Reply:
It's an object ref, the values would not be copied only their lifetime scope extended.

Reply:
Changed in 1ca20d9b5318fc504cede4193b10caa01762a206.